shell.nix

```diff
@@ -22,9 +22,6 @@ let
        # in theory we should only have the above two packages (or better, they should be auto-detected),
        # but due to source-repository-package declarations being considered as local packages by cabal, we need the following packages as well.
        # cf. https://github.com/haskell/cabal/issues/6249 and https://github.com/haskell/cabal/issues/5444
-       ps.cardano-sl-x509
```

Comment:
these changes are not related to this PR.

Reply:
@CodiePP, I need these changes when I work on the node, and basically no one else cares.

Reply:
Dropped, as I finally got https://github.com/input-output-hk/cardano-node/pull/614 reviewed (was hanging since Feb 28).